stl/msbuild/stl_base/stl.files.settings.targets

```diff
@@ -176,6 +176,18 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
         </BuildFiles>
     </ItemGroup>
 
+    <ItemGroup Condition="'$(BuildArchitecture)' == 'i386' or '$(BuildArchitecture)' == 'amd64'">

```

Comment:
Seems like this only fixes this for x86 and x64 but not for arm64?

Reply:
@StephanTLavavej

Reply:
The aliases are not needed on ARM64 IIRC, this is for backwards compatibility on x86/x64 with code that predates ARM64 support. Are you having an issue building some ARM64 code?

Reply:
Yes I have the same LNK2019: unresolved external symbol errors on ARM64 and have applied the AlternateName workaround for arm64 for now.

Reply:
I'd recommend opening a new issue with repro steps then. Are you targetting ARM64EC with some old x64 binary?

Reply:
This is a normal ARM64 static library similarly compiled before this change went in, causing the exact same symptoms after this change when included in a dll targeting ARM64.